Readme.md

```diff
@@ -46,21 +46,21 @@ If you just want to see the whole thing in action, check out the [Samples projec
 
 ## Using Vue.js node packages
 
-The packages will be automatically installed on build (i.e. `dotnet build`) or you can trigger it using the `npm install` command.
+The packages will be automatically installed on build (i.e. `dotnet build`). You can also trigger the installation manually using the `pnpm install` command.
```

Comment:
```suggestion
The packages will be automatically installed on build (i.e. `dotnet build`). You can also trigger the installation manually using the `npm install` command.
```

Reply:
No, `pnpm` is correct. The Node.js Extensions parent library requires [PNPM](https://pnpm.io/), see:
https://github.com/Lombiq/NodeJs-Extensions/#prerequisites